### ISSUE TITLE
Unable to Reproduce Results from Table 2 of your Paper

### ISSUE THREAD

**Issue by oublalkhalid** [google-research/timesfm#96]
Hello,

We are experiencing difficulties in reproducing the results presented in Table 1 of your paper. Despite following all the steps outlined in the methodology, we have not been successful in achieving the expected outcomes.

Could you please assist us in resolving these issues and guide us on how to accurately reproduce the results?

Thank you for your help!

**Comment by rajatsen91** [comment]
I am assuming you mean the finetuning study. If so the notebook at notebooks/finetuning.ipynb can be used to reproduce the study.

The notebook is currently setup to finetune on the whole test set, but we only want 10%. We can just do that by modifying the boundary variables of the train set.

```
dtl = data_loader.TimeSeriesdata(
      data_path=data_path,
      datetime_col="date",
      num_cov_cols=num_cov_cols,
      cat_cov_cols=cat_cov_cols,
      ts_cols=np.array(ts_cols),
      train_range=[int(0.9 * boundaries[0]), boundaries[0]],
      val_range=[boundaries[0], boundaries[1]],
      test_range=[boundaries[1], boundaries[2]],
      hist_len=context_len,
      pred_len=pred_len,
      batch_size=num_ts,
      freq=freq,
      normalize=True,
      epoch_len=None,
      holiday=False,
      permute=True,
  )
```

Note the change above `train_range=[int(0.9 * boundaries[0]), boundaries[0]]`. I also set the `TRAIN_STEPS_PER_EVAL=400` because now one epoch is much shorter. 

I just tested this on `ettm1` with `pred_len=96` and actually I got 0.341 which is actually slightly better than the number we report on Table 2.
